python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -855,6 +855,31 @@ async def arun(self, checkpoint: bool = False) -> None:
             f"Reached maximum iteration ({self.max_iter}) without convergence.\n"
         )
 
+    async def assert_feasibility(self) -> None:
```

Comment:
You used the same code twice... should it be a utility function? Could be part of a future ticket...

Reply:
I honestly don't know. I don't think a utility would be the correct thing to do in this case, because it means we are passing the classes into a method, which is kind of an anti pattern. If I had to swing for the fence here, I would create a Base Script that would have this kind of check in it. Something like having `LatissBaseAlign` inherit `TrackTargetAndTakeImage`.  But yeah, I think we could leave that for a future ticket..